drivers/hmis/spec/requests/hmis/client_search_spec.rb

```diff
@@ -44,16 +45,31 @@
   end
 
   describe 'User access tests' do
-    it 'should only show clients from HMIS data source' do
-      client1 = create :hmis_hud_client, data_source: ds1
-      client2 = create :hmis_hud_client, data_source: ds2
+    let!(:client1) { create :hmis_hud_client, data_source: ds1 }
+    let!(:client2) { create :hmis_hud_client, data_source: ds2 }
+    let!(:client3) { create :hmis_hud_client, data_source: ds1 }
 
+    it 'should only show clients from HMIS data source' do
       response, result = post_graphql(input: {}) { query }
 
       expect(response.status).to eq 200
       clients = result.dig('data', 'clientSearch', 'nodes')
       expect(clients).to include({ 'id' => client1.id.to_s })
       expect(clients).not_to include({ 'id' => client2.id.to_s })
+      expect(clients).to include({ 'id' => client3.id.to_s })
+    end
+
+    it 'should exclude clients enrolled at a project without user view permission' do
```

Comment:
Can you add a test case for WIP enrollments as well? You can make an enrollment WIP by just calling `enrollment.save_in_progress`.